app/assets/stylesheets/_screen.scss

```diff
@@ -657,3 +650,43 @@ div#keycut-help {
 .btn-clipboard {
   padding: 4px 12px 5px;
 }
+
+.activity {
+  @extend .story;
```

Comment:
Thanks! I did not know that `@extend` was not gzip-friendly. o/ I'm working on a mixin.